### ISSUE TITLE
Why minimum of 10 seconds

### ISSUE THREAD

**Issue by h0jeZvgoxFepBQ2C** [james-fray/tab-reloader#145]
Could you please remove this constraint?

I try to monitor an api every 3 or 5 seconds, but I'm not able to, due to this constraint?

**Comment by elandorr** [comment]
Why are you trying to monitor an API via a browser refresh? Just do it in bash.

Circumventing is already possible, as per FAQ: Use shift-click, you can set it to even 1sec.

**Comment by SpyPower** [comment]
Indeed, why 10 seconds min?

I am working with reverse proxies, frontend, backend, azure, and I would love to have this ability. Yeah I can do curls via my bash but, why can't i just open multiple tabs and have auto refresh to find this specific tab which will work when I am making changes?
Why even put a limitation there?

**Comment by h0jeZvgoxFepBQ2C** [comment]
Yeah it would be nice to have it working without pressing shift. 
I also guess the target group of this addon most of the time knows what they are doing.

I like to refresh my JSON api with this gem during testing (i also have another json formatter addon installed), so the json gets autoformatted pretty.

Maybe set the minimum to 1 second, DDOssing with 1 request per second is not really a DDOS imo (if its really the case, you are probably doing something wrong :D)

**Comment by h0jeZvgoxFepBQ2C** [comment]
thanks, but its more handy to just refresh a browser. i dont agree with your DDOS opinion, sorry. Could you add a licence to your github repo?

I will create a fork if you allow it.
